.gitignore

```diff
@@ -7,4 +7,5 @@ packages/testing/*/lib
 coverage/
 tsconfig.build.tsbuildinfo
 yarn-error.log
-integration/rabbitmq/dist
\ No newline at end of file
+integration/rabbitmq/dist
+package-lock.json
```

Comment:
`package-lock.json` must not be ignored, if you remove this change i'll be able to change

Reply:
Resolved